pkg/rpctype/rpc.go

```diff
@@ -101,8 +106,8 @@ func (cli *RPCClient) Close() {
 	cli.c.Close()
 }
 
-func RPCCall(addr string, timeScale time.Duration, method string, args, reply interface{}) error {
-	c, err := NewRPCClient(addr, timeScale)
+func RPCCall(addr string, timeScale time.Duration, useCompression bool, method string, args, reply interface{}) error {
```

Comment:
If there's really only one place where we use it, I wonder if there's sense in keeping this method (that now has even more arguments).

Reply:
You mean to open-code it in syz-manager instead?

Reply:
Yes

Reply:
Done (but I can only build-test this)
PTAL